src/backend/PhotoAPI/API/ServicesConfiguration/SwaggerConfiguration.cs

```diff
@@ -12,7 +16,28 @@ public static void AddSwagger(this IServiceCollection services, IConfiguration c
         {
             services.AddSwaggerGen(conf =>
             {
-                conf.SwaggerDoc(name: "v1", info: new Info() { Title = "Photo API", Version = "v1" });
+                conf.SwaggerDoc(name: "v1", info: new Info()
+                {
+                    Title = "Photo API",
+                    Version = "v1"
+                });
+                conf.AddAuthorization();
+            });
+        }
+
+        private static void AddAuthorization(this SwaggerGenOptions configuration)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'AddAuthorization'.](https://app.codacy.com/manual/iamprovidence/Lama/pullRequest?prid=4910837)